__tests__/app.test.js

```diff
@@ -59,6 +59,59 @@ describe("GET /api/articles", () => {
   });
 });
 
+describe("GET /api/articles?topic=", () => {
+  test("GET:200 responds with an array of article objects, filtered by the topic value specified in the query", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.articles;
+        expect(articles).toHaveLength(1);
+        articles.forEach((article) => {
+          expect(article.topic).toBe("cats");
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number)
```

Comment:
You have two expect statements in here, one is specifically checking that the topic of each article is cats, but the other is checking every other property and also includes the topic which is checking that its any string. This can and should be refactored.